config/routes.rb

```diff
@@ -135,6 +135,8 @@ def scoped_navigation_routes(context, navigation, as_redirects: false)
       resources :tax_returns, only: [] do
         patch "update_certification", to: "tax_returns/certifications#update", on: :member
       end
+      resources :organizations, only: [:index, :create, :new, :edit, :update]
```

Comment:
"except" also exists here, so you could change this to except: :destroy if you felt compelled